webapp/packages/core-blocks/src/Snackbars/Snackbar.tsx

```diff
@@ -62,29 +67,37 @@ export function Snackbar({
     };
   }, []);
 
-  return styled(styles)(
+  return styled(styles)`
+    body-text-block {
+      padding-right: ${!closeAfter && onclose ? '25px' : 'unset'}
+    }
```

Comment:
no matter visible cross mark or not, all notifications should looks the same, remove that